feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -333,6 +336,8 @@ func flapinterface(t *testing.T, dut *ondatra.DUTDevice, port string, action boo
 	dc := gnmi.OC()
 	i := &oc.Interface{}
 	i.Enabled = ygot.Bool(action)
+	i.Type = ethernetCsmacd
```

Comment:
Worth a comment on why setting Type/Name is needed in the 'flap' function.

Reply:
Some vendors require these to be set